datajunction-server/datajunction_server/models/history.py

```diff
@@ -18,14 +18,26 @@ class ActivityType(str, Enum):
     An activity type
     """
 
+    AVAILABILITY = "availability"
     CREATE = "create"
     DEACTIVATE = "deactivate"
     ACTIVATE = "activate"
     UPDATE = "modify"
+
+    # Link columns to dimensions
     LINK = "link"
+    UNLINK = "unlink"
+
     TAG = "tag"
+
+    # Set column attributes
     SET_ATTRIBUTE = "set_attribute"
 
+    # Node materialization
+    ADD_MATERIALIZATION = "add_materialization"
+    UPDATE_MATERIALIZATION = "update_materialization"
+    DELETE_MATERIALIZATION = "delete_materialization"
```

Comment:
Similar to the other comment, I think there should be an entity type `MATERIALIZATION` and then it can be combined accordingly with activity types of `CREATE`, `UPDATE`, or `DELETE`. Also similarly for `AVAILABILITY`